### PR TITLE
add iqtree v2.3.5 cmaple

### PR DESCRIPTION
**Pull request by julienfumey**
add Dockerfiles for iqtree2 v.2.3.5.cmaple

### CODE REVIEW

File: iqtree/v2.3.5.cmaple/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# IQ-TREE
+# http://www.iqtree.org/
+# 2.2.5
```
Comment (fredericlemoine) [inline iqtree/v2.3.5.cmaple/Dockerfile]: May be 2.3.5.cmaple

File: iqtree/v2.3.5.cmaple/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+# base image: Ubuntu
+FROM ubuntu:22.04
```
Comment (fredericlemoine) [inline iqtree/v2.3.5.cmaple/Dockerfile]: We can update ubuntu to 24.04?

File: iqtree/v2.3.5.cmaple/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+# File Author / Maintainer
+LABEL maintainer="frederic.lemoine@pasteur.fr"
```
Comment (fredericlemoine) [inline iqtree/v2.3.5.cmaple/Dockerfile]: You may update the maintainer

**Comment by julienfumey** [comment]
I fixed all requests

**Comment by fredericlemoine** [comment]
The image should be available https://hub.docker.com/r/evolbioinfo/iqtree/tags/ 

`docker pull evolbioinfo/iqtree:v2.3.5.cmaple`

**Pull request merged**